rougelike_game/Map_handler.py

```diff
@@ -0,0 +1,30 @@
+
+looking = ' '
+
+def draw_level(level):
+    for y in range(22):
+        for x in range(22):
+            print(level[y][x], end='')
+        print('')
+
+
+def handle_level(level):
+    level_file = open('Maps/map1.txt', 'r')
+    level_str = level_file.read()
+    rows = level_str.split('\n')
+
+    for row in rows:
+        level.append(list(row))
+
+    return level
+
+
+def find_player(level, Player, looking):
```

Comment:
why do you pass a Player class as an function argument? you should just import a Player module in this module